libminifi/include/c2/protocols/RESTProtocol.h

```diff
@@ -43,7 +43,7 @@ class RESTProtocol : public HeartbeatJsonSerializer {
  protected:
   void initialize(core::controller::ControllerServiceProvider* controller, const std::shared_ptr<Configure> &configure);
   void serializeNestedPayload(rapidjson::Value& target, const C2Payload& payload, rapidjson::Document::AllocatorType& alloc) override;
-  static C2Payload parseJsonResponse(const C2Payload &payload, std::span<const std::byte> response);
+  C2Payload parseJsonResponse(const C2Payload &payload, std::span<const std::byte> response);
```

Comment:
Even if it's no longer `static`, it could still be `const`, as it doesn't seem to modify any members.

Reply:
done